portfolio/src/main/java/com/google/sps/data/Service.java

```diff
@@ -0,0 +1,51 @@
+package com.google.sps.data;
+
+/** Class containing the data for Services. */
+public final class Service {
+
+  private final long id;
+  private final String name;
+  private final String description;
+  private final long provider;
+  private final long averageRating;
+
+  /**
+   * @param {!long} id The unique identifier for each Service.
+   * @param {!String} name The name of this Service.
+   * @param {!String} description The description of this Service.
+   * @param {!long} provider The id of this Service's provider.
+   * @param {!long} averageRating This Service's average Rating.
+   */
+  public Service(long id, String name, String description, long provider, String averageRating) {
```

Comment:
Change property name as listed in database design doc:
i.e changinge id = service_id and provider into user_id or provider_id helps make it more clear.

Reply:
roger that